docs/api/materials/Material.html

```diff
@@ -162,8 +162,8 @@ <h3>[property:String name]</h3>
 
 		<h3>[property:Boolean needsUpdate]</h3>
 		<p>
-		Specifies that the material needs to be updated at the WebGL level.
-		Set it to true if you made changes that need to be reflected in WebGL.<br />
+		Specifies that the material needs to be recompiled.
```

Comment:
maybe just "to be compiled". Is this the only way to compile a material? Even scene.compile() or whatever it is, wouldn't compile without this flag?

Reply:
Recompiled is fine.